packages/office-addin-cli/src/parse_json_script_variable.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+const readJson = require('read-package-json-fast');
+
+export async function parseJSONScriptVariable(variable: string): Promise <string | undefined> {
+    // NPMv7 Support
+    if(process.env.npm_package_json) {
+        const pkg = await readJson(`${process.env.npm_package_json}`);

```

Comment:
So you read and parse the package.json every time?  Maybe there's a better way...

Reply:
Sure, my mistake. Just fixed it so it is only read once!